README.md

```diff
@@ -197,7 +197,7 @@ index.search(
 
 ## 🤖 Compatibility with Meilisearch
 
-This package only guarantees the compatibility with the [version v1.0.0 of Meilisearch](https://github.com/meilisearch/meilisearch/releases/tag/v1.0.0).
+This package guarantees compatibility with [version v1.x of Meilisearch](https://github.com/meilisearch/meilisearch/releases/latest), but some features may not be present. Please check the [issues](https://github.com/meilisearch/meilisearch-python/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+label%3Aenhancement) for more info.
```

Comment:
https://github.com/meilisearch/meilisearch-python/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+label%3Aenhancement brings up no issues, but there are some labeled `enhancement`. I think this link is searching for things labeled with both `good first issue` and `enhancement`.

It may be the case that a new feature isn't labeled `good first issue`, it's also possible an `enhancement` won't be for a previously missing feature specifically from the Meilisearch API. Maybe just link to the issues page and not specific labels?

Reply:
Some SDKs have a lot of issues, so if we don't filter, will be hard for the user to understand which "Meilisearch feature" is missing. But the thing is all of the issues generated during the pre-release period, **will contain both labels**. 

Maybe one good improvement is to adopt a new label, but I don't want to introduce a new label right now.

Reply:
> Maybe one good improvement is to adopt a new label, but I don't want to introduce a new label right now.

I agree if we add this link to all READMEs it might be better to create a specific label for this link like `missing-feature` or something